code/renderers/react/template/stories/ts-argtypes.stories.tsx

```diff
@@ -78,14 +78,14 @@ export const TsComponentProps = { parameters: { component: TsComponentPropsCompo
 
 export const TsJsdoc = { parameters: { component: TsJsdocComponent } };
 
-const addChromaticIgnore = async (element: HTMLElement) => {
-  const row = element.parentElement?.parentElement;
-  if (row?.nodeName === 'TR') {
-    row.setAttribute('data-chromatic', 'ignore');
-  } else {
-    throw new Error('the DOM structure changed, please update this test');
-  }
-};
+// const addChromaticIgnore = async (element: HTMLElement) => {
+//   const row = element.parentElement?.parentElement;
+//   if (row?.nodeName === 'TR') {
+//     row.setAttribute('data-chromatic', 'ignore');
+//   } else {
+//     throw new Error('the DOM structure changed, please update this test');
+//   }
+// };
```

Comment:
?

Reply:
won't merge with this. this is code that makes ignore regions for all the key data that we want to test. will probably delete before the merge & then re-add it if we see test flake